types/.release-it.ts

```diff
@@ -0,0 +1,18 @@
+import type { Config } from 'release-it'
+
+export default {
+  hooks: {
+    'after:bump': 'yarn run build',
+  },
+  git: {
+    commitMessage: 'Release `tangle-substrate-types` v${version}',
+    tag: false,
+    push: false,
+  },
+  github: {
+    release: false,
```

Comment:
A github release too sounds good actually!

Reply:
Sure, I'll add the GitHub release.